### PR TITLE
Add SuggestCommentScore

### PR DESCRIPTION
**Pull request by chrismohrii**
Let's the user submit feedback on any Perspective score

image
image
image

### CODE REVIEW

File: capstone/src/main/java/com/google/sps/servlets/SuggestCommentServlet.java [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+      return;
+    }
+
+    String text = info.getText();
+    String lang = info.getLang();
+    HashMap<String, String> feedback = info.getFeedback();
+    
+    // Make the request to Perspective API
+    OkHttpClient client = new OkHttpClient();
+		String json = makePerspectiveJson(text, lang, feedback);
```
Comment (danieldortega77) [inline capstone/src/main/java/com/google/sps/servlets/SuggestCommentServlet.java]: nit: Let's remove this empty space

File: capstone/src/main/java/com/google/sps/servlets/SuggestCommentServlet.java [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+    commentValue.put("text", text);
+    for (String suggestion : feedback.keySet()) {
+      JSONObject value = new JSONObject();
+      value.put("value", feedback.get(suggestion));
+      JSONObject summaryScoreValue = new JSONObject();
+      summaryScoreValue.put("summaryScore", value);
+      attributesValue.put(suggestion, summaryScoreValue);
+    }
+
```
Comment (danieldortega77) [inline capstone/src/main/java/com/google/sps/servlets/SuggestCommentServlet.java:95]: I think this function could use a comment or two describing how the JSON should be structured, or possibly a general outline of the object we're trying to build (`{ text : __, comment: ___, ... }`). As the code is, it's slightly hard to understand where everything fits.

File: capstone/src/main/java/com/google/sps/servlets/SuggestCommentServlet.java (hunk 1 above)
Comment (chrismohrii) [reply capstone/src/main/java/com/google/sps/servlets/SuggestCommentServlet.java]: Haha quick review! I think this is fixed in "Fix indentation"

File: capstone/src/main/java/com/google/sps/servlets/SuggestCommentServlet.java (hunk 2 above)
Comment (chrismohrii) [reply capstone/src/main/java/com/google/sps/servlets/SuggestCommentServlet.java:95]: Agreed - I added a comment of that form

File: capstone/src/main/webapp/script.js [hunk 3]
```diff
@@ -461,7 +461,7 @@ function colorSubstring(substringEl, toxicityScore) {
 }
 
 /** Creates a tooltip for a substring in the detailed analysis */
-function createTooltip(attributes) {
+function createTooltip(attributes, text, lang,requestedAttributes) {
```
Comment (olu144) [inline capstone/src/main/webapp/script.js]: nit: lets add a space after each comma

Comment (chrismohrii) [reply capstone/src/main/webapp/script.js]: Fixed

**Pull request closed without merging**